invenio_accounts/models.py

```diff
@@ -141,8 +142,14 @@ def __init__(self, *args, **kwargs):
         """Constructor."""
         user_profile = kwargs.pop("user_profile", {})
         preferences = kwargs.pop("preferences", {})
-        preferences.setdefault("visibility", "restricted")
-        preferences.setdefault("email_visibility", "restricted")
+        preferences.setdefault(
+            "visibility",
+            current_app.config.get("ACCOUNTS_USER_VISIBILITY", "restricted"),
```

Comment:
would it make sense to add this variable to the config.py instead of defining the default here?
it would be more visible if exposed in config.py rather than hardcoded here

Reply:
I would also rename it to 
```suggestion
            current_app.config.get("ACCOUNTS_DEFAULT_USER_VISIBILITY", "restricted"),
```

Can you also add some tests for this?

Reply:
@zzacharo It appears that the functionality has already undergone testing. However, when I provide non-standard input values like "restricted" or "public", some existing tests fail. Can you confirm that I am not introducing additional functionality, and if not, can you provide examples of similar tests?

Reply:
@kpsherva  Certainly, I will take care of it.